modules/framework/test.py

```diff
@@ -0,0 +1,65 @@
+from modules.db.db import Db
+from modules.framework.timer import Timer
+from modules.framework.parser import DataProvider
+from modules.db.model.car import Car
+from pathlib import Path
+import json
+
+
+class Test:
+    def __init__(self, db: Db, data_file: str, results_filename: str):
```

Comment:
maybe its better to use dependency injection for data_file - should Test class be aware of data path or should it take DataProvider object as parameter